AztecTests/HTML/ElementNodeTests.swift

```diff
@@ -721,7 +722,7 @@ class ElementNodeTests: XCTestCase {
 
         let wrapRange = NSRange(location: text1.characters.count, length: text2.characters.count)
 
-        paragraph.forceWrap(range: wrapRange, inNodeNamed: "b", withAttributes: [], equivalentElementNames: [])
+        paragraph.forceWrap(range: wrapRange, inElement: ElementNodeDescriptor(name: "b"))
```

Comment:
We could use this PR to refactor the use of "b" for StandardElementType.b.rawValue. Just to get rid of this loose strings :)